tests/foreman/cli/test_activationkey.py

```diff
@@ -815,8 +816,23 @@ def test_positive_delete_manifest(module_org):
     :expectedresults: Deleting a manifest removes it from the Activation
         key
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
     """
+    with manifests.clone() as manifest:
+        upload_manifest(function_org.id, manifest.content)
+    new_ak = make_activation_key({'organization-id': function_org.id})
+    subscription_result = Subscription.list(
+        {'organization-id': function_org.id, 'order': 'id desc'}, per_page=False
+    )
+    result = ActivationKey.add_subscription(
+        {'id': new_ak['id'], 'subscription-id': subscription_result[-1]['id']}
+    )
+    assert 'Subscription added to activation key.' in result
+    Subscription.delete_manifest({'organization-id': function_org.id})
+    ak_subs_info = ActivationKey.subscriptions(
+        {'id': new_ak['id'], 'organization-id': function_org.id}
+    )
+    assert len(ak_subs_info) == 4
```

Comment:
WIll the ak_subs_info always be 4 ?

Reply:
Yeah this number looks like pure magic, I'd at least like to see some in-line comment explaining it.  I think it would be a lot more consistent to compare it to a value read before the manifest is deleted, or an explicit check that a specific AK subscription is removed instead of just validating the count of subscriptions.

Reply:
But, yes @ColeHiggins2 @mshriver it will be better to compare to a value read before the manifest is deleted. I will update that.